pkg/image/validation/kubernetes.go

```diff
@@ -14,10 +16,13 @@ import (
 )
 
 const (
-	k8sComponent = "Kubernetes"
-	httpScheme   = "http"
-	httpsScheme  = "https"
-	ociScheme    = "oci"
+	k8sComponent     = "Kubernetes"
+	httpScheme       = "http"
+	httpsScheme      = "https"
+	ociScheme        = "oci"
+	k8sDir           = "kubernetes"
+	k8sConfigDir     = "config"
+	serverConfigFile = "server.yaml"
```

Comment:
Try to reuse the constants defined somewhere else -- we don't want to run into the case of some of these getting modified in one place but not the other.